packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/multiplier_take_profit.js

```diff
@@ -0,0 +1,72 @@
+import { localize } from '@deriv/translations';
+import { getCurrencyDisplayCode } from '@deriv/shared';
+import { config } from '../../../../constants/config';
+
+Blockly.Blocks.multiplier_take_profit = {
+    init() {
+        this.jsonInit(this.definition());
+        this.setMovable(false);
+        this.setDeletable(false);
```

Comment:
I don't know what the convention we follow in DBot but if we're trying to use this as `is_movable` and `is_deletable` maybe we should use `setIsDeletable` and `setIsMovable`?

Reply:
`setMovable` and `setDeletable` are blockly methods.